core/src/main/resources/mcmod.info

```diff
@@ -69,5 +69,18 @@
     "logoFile": "/assets/genetics/logo.png",
     "parent": "",
     "dependencies": [ "BinnieCore" ]
+  },
+  {
+    "modid": "binniedesign",
+    "name": "Binnie's Design",
+    "description": "Adds a lot of blocks with different textures.",
```

Comment:
Binnie's Design is actually just a library to support those features in ExtraTrees and Botany, it does not add anything itself.